File: sorting and searching/selectionsort.cpp
#include<iostream>
using namespace std;
void selectionsort(int arr[],int n){
    for(int i=0;i<n;i++){
        int min=i;
        for(int j=i+1;j<n;j++){
            if(arr[j]<arr[min])
                min=j;
        }
        int temp=arr[i];
        arr[i]=arr[min];
        arr[min]=temp;
    }
} 
int main(){
    int i,n;
    cout<<"Enter the size of the array:";
    cin>>n;
    int arr[n];
    cout<<"Enter the elements of the array:";
    for(i=0;i<n;i++){
        cin>>arr[i];
    }
    selectionsort(arr,n);
    cout<<"Sorted array in Ascending order by using Selction Sort:";
    for(i=0;i<n;i++){
        cout<<" "<<arr[i]<<" ";
    }
    cout<<"\n";
}

File: sorting and searching/selectionsort.py
#selection sort in python3
def selectionsort(arr,n):
    for i in range(n):
        min=i   
        for j in range(i+1,n):
            if(arr[j]<arr[min]):
                min=j
        arr[i],arr[min]=arr[min],arr[i] #swaping the minimum element and element at beginning
n=int(input("Enter the size of the array:"))
arr=[]
print("Enter the elements of the array:")
for i in range(n):
    ele=int(input())
    arr.append(ele)
selectionsort(arr,n)
print("Sorted array in Ascending order by using Selection Sort:")
print(arr)
